### PR TITLE
Take keeper fees into account in isMarginLiquidatable

### PR DESCRIPTION
**Pull request by 0xjocke**

### CODE REVIEW

File: markets/bfp-market/contracts/storage/Margin.sol [hunk 1]
```diff
@@ -382,6 +409,15 @@ library Margin {
 
         // Ensure that there is collateralUsd on the account to ensure this account margin can be liquidated.
         Margin.MarginValues memory marginValues = Margin.getMarginUsd(accountId, market, price);
-        return marginValues.discountedMarginUsd == 0 && marginValues.collateralUsd != 0;
+
+        return
+            marginValues.discountedMarginUsd.toInt() -
+                getMarginLiquidationOnlyReward(
```
Comment (davidvuong) [inline markets/bfp-market/contracts/storage/Margin.sol:415]: As mentioned this deduction of `getMarginLiquidationOnlyReward` also needs to be included in `modifyCollateral`

Comment (0xjocke) [reply markets/bfp-market/contracts/storage/Margin.sol:412]: Yep thanks! Addressed

File: markets/bfp-market/contracts/modules/LiquidationModule.sol [hunk 2]
```diff
@@ -314,42 +314,22 @@ contract LiquidationModule is ILiquidationModule {
...
-
-        if (!Margin.isMarginLiquidatable(accountId, market, oraclePrice)) {
+        Margin.MarginValues memory marginValues = Margin.getMarginUsd(
+            accountId,
+            market,
+            oraclePrice
+        );
+        if (
+            !Margin.isMarginLiquidatable(accountId, market, marginValues) ||
+            marginValues.collateralUsd == 0
```
Comment (davidvuong) [inline markets/bfp-market/contracts/modules/LiquidationModule.sol:331]: Minor: You could probably put `marginValues.collateralUsd == 0` before the `!Margin.isMarginLiquidatable(accountId, market, marginValues)` to be a little bit more efficient.

File: markets/bfp-market/contracts/modules/LiquidationModule.sol [hunk 3]
```diff
@@ -314,42 +314,22 @@ contract LiquidationModule is ILiquidationModule {
...
-        return MathUtil.min(liqFeeWithRewardInUsd, globalConfig.maxKeeperFeeUsd);
-    }
-
     /// @inheritdoc ILiquidationModule
     function liquidateMarginOnly(uint128 accountId, uint128 marketId) external {
         FeatureFlag.ensureAccessToFeature(Flags.LIQUIDATE_MARGIN_ONLY);
 
         Account.exists(accountId);
         PerpMarket.Data storage market = PerpMarket.exists(marketId);
         uint256 oraclePrice = market.getOraclePrice();
```
Comment (davidvuong) [inline markets/bfp-market/contracts/modules/LiquidationModule.sol:323]: `oraclePrice` doesn't seem to be used anywhere else so you can avoid the var here.

**Review by davidvuong — APPROVED** [review]
Minor comments but LGTM

**Pull request merged**